Formula/qt.rb

```diff
@@ -7,6 +7,7 @@ class Qt < Formula
   mirror "https://mirrors.dotsrc.org/qtproject/archive/qt/5.15/5.15.0/single/qt-everywhere-src-5.15.0.tar.xz"
   mirror "https://mirrors.ocf.berkeley.edu/qt/archive/qt/5.15/5.15.0/single/qt-everywhere-src-5.15.0.tar.xz"
   sha256 "22b63d7a7a45183865cc4141124f12b673e7a17b1fe2b91e433f6547c5d548c3"
+  revision 1
```

Comment:
I don't think the `revision` is necessary; that'll force a reinstall for everyone and, given this is a built-time error, anyone who needs this patch won't have the previous version of `qt` installed.

```suggestion
```